webToDatabaseFunctions.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**to pull subtitles from about_me table in db
+ *
+ * @param $db array subtitle column from about_me table
+ * @param $id int id column from about_me table
+ *
+ * @return mixed subtitle info from db
+ */
+function getSubtitleFromDB($db, $id) {
+    $query = $db->prepare("SELECT `subtitle` 
+                            FROM `about_me`
+                            WHERE `id` = " . $id . "
+                            GROUP BY `subtitle`;");
+    $query->execute();
+    return $query->fetchAll();
```

Comment:
If we're only getting a single response we can use a fetch() rather than fetchall(), reduces the time spent with database (i think)

Reply:
fetch all breaks the code (!?!) so I'm going to leave fetch All in for now...!

Reply:
oh no! Might be worth var_dump ing its result (if you want to use fetch()) to see where it's going wrong